### PR TITLE
Documentation update Fixes #27

### LINKED ISSUE #27: Doc Update: Slack need to invite the bot into the channel
### Environment
* Python version: n/a
* Nautobot version:  1.0.0b2
* nautobot-plugin-chatops version: 1.0.0

### Steps to Reproduce
1. Followed setup instructions
2. Troubleshot error with the bot
3. Found that the bot needed to be invited in

### Expected Behavior

Would expect that this would be in the documentation

### Observed Behavior

The bot was not fully usable with the docs as is

### PR DESCRIPTION
**Pull request by dgjustice**
- Moved all the install text to the chat_setup document
- Formatting changes to make mdlint happy
- Add a couple images for eye candy
- Add a note about inviting the chatbot in Slack

Fixes #27

### CODE REVIEW

File: docs/chat_setup.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+```
+
+Nautobot supports `Slack`, `MS Teams` and `Webex Teams` as backends but by default all of them are disabled. You need to explicitly enable the chat platform(s) that you want to use in the `PLUGINS_CONFIG` with one or more of `enable_slack`, `enable_ms_teams` or `enable_webex_teams`.
+
+The plugin behavior can be controlled with the following list of general settings:
+
+| Configuration Setting        | Description | Mandatory? | Default |
+| ---------------------------- | ----------- | ---------- | ------- |
+| `delete_input_on_submission` | After prompting the user for additional inputs, delete the input prompt from the chat history | No | `False` |
+
```
Comment (jvanderaa) [inline docs/chat_setup.md:34]: I'd like to see this left where it was on the main page, keeping the plugin documentation consistent on the install instructions personally. I'm also ok if this moves to here.

This is something else that I'm noticing as part of this PR is that `Mattermost` is missing from the list of backends, which is now supported.

**Review by jvanderaa — CHANGES_REQUESTED** [review]
At a minimum the one change to include `Mattermost`, if not the other about leaving installation.

File: docs/chat_setup.md (hunk 1 above)
Comment (jvanderaa) [reply docs/chat_setup.md:34]: I think we have enough present. My original comment was to keep install locally in the main readme. My mind is changed on that front!

Comment (dgjustice) [reply docs/chat_setup.md:34]: Can we have the author of the Mattermost change update the documentation appropriately?  I have never used the platform and know nothing of the setup.

Comment (jvanderaa) [reply docs/chat_setup.md:34]: I'll double check on the setup if it is there. But the front page on line 27 of the readme should have that update.

Comment (dgjustice) [reply docs/chat_setup.md:34]: Oh, I see what you are saying now.

**Review by jvanderaa — APPROVED** [review]

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Let's remove the install instructions from primary readme since it is moved to the install items.

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**